graylog2-server/src/main/java/org/graylog2/bootstrap/CmdLineTool.java

```diff
@@ -283,7 +283,7 @@ public void doRun(Level logLevel) {
 
         coreConfigInjector = setupCoreConfigInjector();
 
-        final Set<Plugin> plugins = loadPlugins(getPluginPath(configFile), chainingClassLoader);
+        final Set<Plugin> plugins = Sets.union(BundledPlugins.BUNDLED_PLUGINS, loadPlugins(getPluginPath(configFile), chainingClassLoader));
```

Comment:
Does this work without having `hashCode` and `equals` implemented in plugin classes?